Switch.cpp

```diff
@@ -0,0 +1,37 @@
+#ifndef Switch_cpp
+#define Switch_cpp
+#include <iostream>
+#include "Enum.cpp"
+#include "Draw.cpp"
+#include "Square.cpp"
+#include "Rectangle.cpp"
+#include "Circle.cpp"
+
+using namespace std;
+
+Draw* selectShape(const string& shape)
```

Comment:
The name of the method is deceiving, it's creating a Shape type but the method says `select`.